src/scss/components/_infoSection.scss

```diff
@@ -0,0 +1,60 @@
+.info {
+    margin: 40px 20px;
```

Comment:
As a recommendation to create a standard, you should consider to use bootstrap classes like `class= "container"` for responsive margins.